platform/android/export/export_plugin.cpp

```diff
@@ -2343,15 +2348,18 @@ Error EditorExportPlatformAndroid::sign_apk(const Ref<EditorExportPreset> &p_pre
 	String keystore;
 	String password;
 	String user;
+	String userPassword;
```

Comment:
```suggestion
	String user_password;
```

Godot uses `snake_case` in C++. The same applies to other variable names below.

Reply:
already edited.